docs/general/ddl/partitioned-tables.rst

```diff
@@ -17,24 +17,70 @@ Partitioned tables
 Introduction
 ============
 
-A partitioned table is a virtual table that can be created by naming one or
-more columns by which it is split into separate internal tables, called
-*partitions*.
+A partitioned table is a virtual table consisting of zero or more partitions. A
+partition is similar to a regular single table and consists of one or more
+shards.
 
-When a record with a new distinct combination of values for the configured
-:ref:`partition columns <gloss-partition-column>` is inserted, a new partition
-is created and the document will be inserted into this partition.
+::
+
+    partitioned_table
+      |
+      +-- partition 1
+      |     |
+      |     +- shard 0
+      |     |
+      |     +- shard 1
+      |
+      +-- partition 2
+            |
+            +- shard 0
+            |
+            +- shard 1
+
+
+A table becomes a partitioned table by defining :ref:`partition columns
+<gloss-partition-column>`.  When a record with a new distinct combination of
+values for the configured :ref:`partition columns <gloss-partition-column>` is
+inserted, a new partition is created and the document will be inserted into
+this partition.
+
+A partitioned table can be queried like a regular table. 
+
+Partitioned tables have the following advantages:
+
+- The number of shards can be changed on the partitioned table, which will then
+  change how many shards will be used for the next partition creation. This
+  enables one to start out with few shards per partition initially, and scale
+  up the number of shards for later partitions once traffic and ingest rates
+  increase with the lifetime of an application.
+
+- Partitions can be backed up and restored individually.
+
+- Queries which contain filters in the ``WHERE`` clause which identify a single
+  partition or a subset of partitions is less expensive than querying all
+  partitions because the shards of the excluded partitions won't have to be
+  accessed.
+
+- Deleting data from a partitioned table is cheap if the optimizer identifies
+  that a full partition can be dropped.
+
+
+Partitioned tables have the following disadvantages:
+
+
+- If the partitioned columns are badly chosen you can end up with too many
```

Comment:
```suggestion
- If the partition columns are badly chosen you can end up with too many
```

Minor consistency change to reflect the usage at the beginning of the Introduction.

`partitioned columns` is fine too but according to Glossary
> **Partitioned columns** Better known as a partition column.
> https://crate.io/docs/crate/reference/en/master/appendices/glossary.html#gloss-partitioned-column